common/web/gesture-recognizer/src/engine/touchEventEngine.ts

```diff
@@ -86,6 +82,19 @@ export class TouchEventEngine<HoveredItemType, StateToken = any> extends InputEv
     }
   }
 
+  public unlockTouchpoint? = (touchpoint: GestureSource<HoveredItemType, StateToken, GesturePath<HoveredItemType, StateToken>>) => {
+    const lock = this.inputStartSignalMap.get(touchpoint);
+    if(lock) {
+      lock.resolve();
+      this.inputStartSignalMap.delete(touchpoint);
```

Comment:
Why do you remove the item from the map after the lock is resolved? That seems backwards from most locking strategies?

Reply:
Right... I may have gotten a tad used to how JS works in this regard.  JS is only ever single-threaded, so I was kind of abusing that here, huh?  I've gone ahead and adjusted the pattern to match what would be needed if actual concurrency were possible.